adb_graphics/default_specs.yml

```diff
@@ -485,21 +657,41 @@ cloudcover:
     unit: '%'
   low:
     <<: *cld_cover
+    cfgrib:
+      shortName: lcc
+      typeOfLevel: lowCloudLayer
+      level: 0
     ncl_name: LCDC_P0_L214_{grid}
     title: Low-Level Cloud Cover
   mid:
     <<: *cld_cover
+    cfgrib:
+      shortName: mcc
+      typeOfLevel: middleCloudLayer
+      level: 0
     ncl_name: MCDC_P0_L224_{grid}
     title: Mid-Level Cloud Cover
   total:
     <<: *cld_cover
+    cfgrib:
+      hrrr:
+        shortName: tcc
+        typeOfLevel: atmosphere
+        stepType: instant
+        level: 0
+      rrfs:
+        shortName: tcc
+        typeOfLevel: atmosphereSingleLayer
+        stepType: instant
+        level: 0
     ncl_name: TCDC_P0_L{level_type}_{grid}
     title: Total Cloud Cover
 clwmr: # Cloud water Mixing Ratio
   ua:
-    ncl_name: 
-      nat: CLWMR_P0_L105_{grid}
-      prs: CLWMR_P0_L100_{grid}
+    cfgrib:
+      shortName: clwmr
+      typeOfLevel: '{{ "isobaricInhPa" if file_type == "prs" else "hybrid" }}'
```

Comment:
This is another nice way to use uwtools Jinja2 expressions...we don't need duplicate sections, and subsequent Python processing to see if the prs/nat files have different values for the same model.